### ISSUE TITLE
random MPI crashes in CI

### ISSUE THREAD

**Issue by cyrush** [LLNL/conduit#994]
we are seeing occasional MPI test crashes in our python windows CI cases. 

Where there is smoke - there is fire.

Here is an example of one of the crashes:

```
2022-08-11T19:07:34.4442651Z    97/125 Test  #97: t_blueprint_mpi_mesh_relay ................................***Failed    0.06 sec
2022-08-11T19:07:34.4510436Z EXEC : error : Failed to post close command error 1717 [D:\a\conduit\conduit\build\RUN_TESTS.vcxproj]
2022-08-11T19:07:34.4520973Z EXEC : error : unable to tear down the job tree. exiting... [D:\a\conduit\conduit\build\RUN_TESTS.vcxproj]
```

(I will add more as we see them)

It could be an MPI configuration issue, or maybe a memory error.

**Comment by cyrush** [comment]
```

   93/125 Test  #93: t_blueprint_mpi_smoke .....................................***Failed    0.06 sec
EXEC : error : Failed to post close command error 1726 [D:\a\conduit\conduit\build\RUN_TESTS.vcxproj]
EXEC : error : unable to tear down the job tree. exiting... [D:\a\conduit\conduit\build\RUN_TESTS.vcxproj]
  
          Start  94: t_blueprint_mpi_mesh_query
   94/125 Test  #94: t_blueprint_mpi_mesh_query ................................   Passed    0.07 sec
          Start  95: t_blueprint_mpi_mesh_transform
   95/125 Test  #95: t_blueprint_mpi_mesh_transform ............................   Passed    0.09 sec
          Start  96: t_blueprint_mpi_mesh_verify
   96/125 Test  #96: t_blueprint_mpi_mesh_verify ...............................   Passed    0.07 sec
          Start  97: t_blueprint_mpi_mesh_relay
   97/125 Test  #97: t_blueprint_mpi_mesh_relay ................................***Failed    0.06 sec
EXEC : error : Failed to post close command error 1726 [D:\a\conduit\conduit\build\RUN_TESTS.vcxproj]
EXEC : error : unable to tear down the job tree. exiting... [D:\a\conduit\conduit\build\RUN_TESTS.vcxproj]
  

 The following tests FAILED:
  	 93 - t_blueprint_mpi_smoke (Failed)
  	 97 - t_blueprint_mpi_mesh_relay (Failed)
```

**Comment by cyrush** [comment]
the fact that t_blueprint_mpi_smoke fails, makes me suspicious of a resource issue in the CI env.

**Comment by cyrush** [comment]
```
   94/125 Test  #94: t_blueprint_mpi_mesh_query ................................***Failed    0.06 sec
EXEC : error : Failed to post close command error 1726 [D:\a\conduit\conduit\build\RUN_TESTS.vcxproj]
EXEC : error : unable to tear down the job tree. exiting... [D:\a\conduit\conduit\build\RUN_TESTS.vcxproj]
```

**Comment by cyrush** [comment]
mystery solved:

The github actions setup mpy4py action we are using is tested with windows-2019.

Our clang build was pinned to that, but other windows cases were using `windows-latest`, which recently changed to `windows-2022`. 

To resolve, we are pinning to windows-2019 in #995

**Comment by cyrush** [comment]
resolved in #995
